src/app/code/community/EasyTranslate/Connector/Model/Resource/Project.php

```diff
@@ -290,4 +290,14 @@ protected function _lookupTargetStoreIds(int $id): array
 
         return $adapter->fetchCol($select, $binds);
     }
+
+    private function randomiseString()
+    {
+        $strong_result = true;
```

Comment:
Please use camel case as always.

I know PhpStorm complains, but if we do not use the return value at all, I do not see value in having this. So either remove it or if you think we should keep it, move it down to the openssl call.

Reply:
I will keep it since i dont want PhpsStorm to complain.